### PR TITLE
[WIP]Restore Japanese resources.

### PR DESCRIPTION
**Pull request by sot528**

### CODE REVIEW

**Comment by fiestared** [comment]
@sot528 I added japanese resource :) please check it!

File: src/yaml/ja.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  token:
+    header: 良質な情報の発掘
+    body: |
+      多くの人々が価値を認めるコンテンツを投稿・いち早く評価した人に対して、
+      より多くのALISトークンを配布します。
+  platform:
+    header: プラットフォーム価値の還元
+    body: |
+      ALISプラットフォームが評価されるほどALISトークンの価値も向上し、貢献した人々に還元されます。
+      すでにSTEEM(https://steem.io)が上述のエコシステムを構築しており、300百億以上の評価を得ています。
```
Comment (sot528) [inline src/yaml/ja.yml]: [nits]
~300百億以上~ => 300億円以上

File: src/yaml/ja.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+cover:
+  caption_line_one: 報酬によって信頼できる記事と人々を明らかにする
+  caption_line_two: ALISは日本初のブロックチェーン技術を用いたソーシャルメディアプラットフォームです
+  caption_line_three: ALISのトークンセールは9月1日 11:00（日本時間）にスタートします
+service_information:
+  header: What is ALIS?
+  body1: |
+    ALISは信頼性の高い情報・人に素早く出会え、
+    誰もが報酬を得ることができるソーシャルメディアプラットフォームです。
+    私たちは類似サービスのSTEEM(https://steem.io)が数百億の評価を得ていることに感銘をうけ、
```
Comment (sot528) [inline src/yaml/ja.yml]: [IMO]
~数百億~ => 300億円以上

To fits other line.

**Comment by fiestared** [comment]
@sot528 fixed it :)

**Comment by sot528** [comment]
@fiestared 
LGTM!
Can you add me to reviewer?

**Pull request merged**